### ISSUE TITLE
Schreibfehler?

### ISSUE THREAD

**Issue by canonknipser** [phpbb-de/wiki#7]
https://github.com/phpbb-de/wiki/blob/f9bce2ed6c46620bb0974697dbbe4107f7017276/controller/main.php#L132-L133

detele statt delete?

**Comment by Crizz0** [comment]
Ja, da gibt es so einige. Ka, ob es dann nicht läuft oder ob der Fehler sich durchzieht.
